### DISCUSSION TITLE
value of the XDG_CURRENT_DESKTOP env

### DISCUSSION THREAD

**Discussion by LuckyTurtleDev** [BuddiesOfBudgie/budgie-desktop#229] [Q&A] [answered]
Which value does the `$XDG_CURRENT_DESKTOP` environment variable has on budgie?

`echo $XDG_CURRENT_DESKTOP`

**Comment by JoshStrobl** [comment, marked as answer]
We report as `Budgie:GNOME` (GNOME part for compatibility reasons)

**Reply by LuckyTurtleDev** [reply]
Thanks
